occurrences/models.py

```diff
@@ -454,6 +456,9 @@ class Enrolment(models.Model):
         verbose_name=_("status"),
         max_length=255,
     )
+    verification_tokens = GenericRelation(
+        VerificationToken, related_query_name="enrolment"
```

Comment:
Enrolment has a field called verification_tokens, which can include all kind of verification tokens (currently there is only 1 available, which is CANCELLATION).